app/src/androidTest/java/org/mozilla/focus/utils/AndroidTestUtils.java

```diff
@@ -132,6 +132,14 @@ public static void tapSettingButton() {
         onView(allOf(withId(R.id.menu_preferences), isDisplayed())).perform(click());
     }
 
+    public static void tapPrivateButtonInMenu() {
+        onView(allOf(withId(R.id.btn_private_browsing), isDisplayed())).perform(click());
```

Comment:
there are two view with the same id : btn_private_browsing
not sure if that's the cause